### ISSUE TITLE
LTX 2.3 OOM on 5090 for lora with 2 images...

### ISSUE THREAD

**Issue by Lu-Gru** [ostris/ai-toolkit#795]
Hello,
Updated today to latest version -> started training lora, i2v / t2v (tested both), and OOM.
Reduced image count to 2 for training dataset, enabled all caches, used 512x512, and OOM... on platform with 32GB VRAM / 96 GB RAM...

It seams like with every update ai-toolkit getting worse...

**Comment by Lu-Gru** [comment]
Strangely job that I run two days ago, with 50+video / no audio / dataset, works.
Cloned that workflow -> changed dataset and pointed to one with 2 images -> OOM ...

Does ai-toolkit support LTX 2.3 Lora training using images ???

**Comment by Lu-Gru** [comment]
Update: got it working...
Had to set `Text Encoder Offload %` to 100% and it started to work.

**Comment by 5andysalive** [comment]
it's not really solved. This is a workaround that wasn't needed before and shouldn't on  enough-vram settings.

**Comment by Lu-Gru** [comment]
> it's not really solved. This is a workaround that wasn't needed before and shouldn't on enough-vram settings.

The main question is how much of VRAM is enough?
According to [LTX-2 manual](https://docs.ltx.video/open-source-model/ltx-2-trainer/ltx-2-training#requirements):
`Nvidia H100 GPU with 80GB+ VRAM - Lower VRAM may work with gradient checkpointing and reduced resolutions`

So we can all thank to AMD/NVidia for selling us "high end" 32GB GPUs... for a kidney, luckily I got another one, so all good :).
